packages/dids/readme.md

```diff
@@ -0,0 +1,37 @@
+
+# DID Manager
+
+DID manager for the dock wallet sdk
```

Comment:
either define DIDs or link to our API docs where we have definitions of DIDs, etc. (or both)

Reply:
Fixed